### DISCUSSION TITLE
请教下大佬如何在查询UID时候优先查询已绑定cookie的数据

### DISCUSSION THREAD

**Discussion by ID114514** [Arondight/Adachi-BOT#446] [Q&A] [answered]
cookie在该项目里大佬写的实时便笺时候已经绑定好了，如何调用实时便笺绑定的cookie用于查询自己数据呢？

**Comment by Arondight** [comment, marked as answer]
1. 不支持 cookie 收集
2. 这样做是不好的，至少已经是侵犯了他人权益

**Reply by mark9804** [reply]
这个不是我们项目自有的功能，你可能要去问问写这个插件的开发者

**Reply by ID114514** [reply]
> 

啊这，不是项目的吗，我是在此项目里看到的 #301

**Reply by mark9804** [reply]
> 我是在此项目里看到的 #301

但是这个PR最终没有合并进来，所以我们项目也没有把cookie和某个用户关联的功能

**Reply by ID114514** [reply]
> > 我是在此项目里看到的 #301
> 
> 但是这个PR最终没有合并进来，所以我们项目也没有把cookie和某个用户关联的功能

所以项目不考虑cookie查询的吗...现在uid查询只显示8角色了很难受，想查自己角色都查不了...

**Reply by Arondight** [reply]
我会考虑，但不是现在，另外你可以直接使用 https://github.com/ixCiel/Adachi-BOT/tree/myBranch ，相关问题去问这个老大哥

**Reply by ID114514** [reply]
我的想法是：cookie一般都是自愿绑定的吧，而且cookie在普通用户手里也不懂得利用，，，反倒是恶意收集利用cookie的估计能自己写插件了吧，他们或许根本不用这个项目的更新内容，反倒是卡住了依赖更新内容的普通用户...

**Reply by Arondight** [reply]
我们无法对 cookie 不被滥用做出任何承诺，因为这是一个开源项目，现在还不到考虑要不要去做恶的时候，至少目前还能查八个角色，而且也有人做这个功能的 fork 啊，用户根本不会被卡住，完全可以使用 fork 来增加这个功能并且保留此项目的全部功能
